_posts/blog/2021-01-06-goodbye-henry.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: "Goodbye, Henry"
+excerpt: "Thank you for all your contributions!"
+categories: blog
+Tags: staff
+date: 2021-01-06
+author: antonie
+---
+
+After two years as the Zcash Foundation's Principal Cryptographic Researcher, [Henry de Valence](https://twitter.com/hdevalence) is stepping down.
+
+Henry was the Zcash Foundation's [first technical hire]((https://www.zfnd.org/blog/henry-de-valence/) ) and left a deep imprint on our work.  He designed the architecture of [Zebra](https://github.com/ZcashFoundation/zebra), the Zcash Foundation's independent implementation of the Zcash protocol and consensus rules.  Zebra has a modern Rust codebase, is memory-safe and multithreaded, and uses an asynchronous verification architecture to allow transparent batch verification of proof data.  Because Zebra is written as a collection of modular component libraries, it will be the foundation of our future engineering projects.
```

Comment:
```suggestion
Henry was the Zcash Foundation's [first technical hire](https://www.zfnd.org/blog/henry-de-valence/) and left a deep imprint on our work.  He designed the architecture of [Zebra](https://github.com/ZcashFoundation/zebra), the Zcash Foundation's independent implementation of the Zcash protocol and consensus rules.  Zebra has a modern Rust codebase, is memory-safe and multithreaded, and uses an asynchronous verification architecture to allow transparent batch verification of proof data.  Because Zebra is written as a collection of modular component libraries, it will be the foundation of our future engineering projects.
```